backend/src/api/books/books.controller.ts

```diff
@@ -0,0 +1,54 @@
+import { BadRequestException, Body, Controller, Get, Post, Query } from '@nestjs/common'
+import { ApiTags, ApiQuery } from '@nestjs/swagger'
+
+import { CreateBookDto } from './dto/createBook.dto'
+import { BooksService } from './books.service'
+
+@ApiTags('books')
+@Controller('/api/books')
+export class BooksController {
+  constructor(private booksService: BooksService) {}
+
+  @Get('/')
+  @ApiQuery({ name: 'limit', required: false, type: Number })
+  @ApiQuery({ name: 'page', required: false, type: Number })
```

Comment:
If we have a search pattern parametter. What we will do for this API books, now?